android/src/com/frostwire/android/core/ConfigurationManager.java

```diff
@@ -62,13 +65,24 @@ public static ConfigurationManager instance() {
     private ConfigurationManager(Application context) {
         preferences = PreferenceManager.getDefaultSharedPreferences(context);
         editor = preferences.edit();
-
         defaults = new ConfigurationDefaults();
         initPreferences();
+        migrateWifiOnlyPreference();
+    }
+
+    /**
+     * This method migrates the deprecated {@link Constants#PREF_KEY_NETWORK_USE_MOBILE_DATA}
+     * to the new {@link Constants#PREF_KEY_NETWORK_USE_WIFI_ONLY}
+     * it will be run only once
+     */
+    private void migrateWifiOnlyPreference() {
+        final String MIGRATION_MARKER = "frostwire.prefs.migration.use_mobile_data.use_wifi_only";
+        if(!getBoolean(MIGRATION_MARKER)) {
+            setBoolean(Constants.PREF_KEY_NETWORK_USE_WIFI_ONLY, !getBoolean(Constants.PREF_KEY_NETWORK_USE_MOBILE_DATA));
+            setBoolean(MIGRATION_MARKER, true);
```

Comment:
Here's a [PR](https://github.com/grzesiekrzaca/frostwire/pull/6), let's get this over with